core/templates/pages/creator-dashboard-page/modal-templates/upload-activity-modal.controller.spec.ts

```diff
@@ -53,9 +53,11 @@ describe('Upload Activity Modal Controller', function() {
       size: 100,
       name: 'file.mp3'
     };
-    // The document has strict type rules which will fail typescript lint tests
-    // without the @ts-ignore.
-    // @ts-ignore
+    // TODO(#10113): Refactor the test to not use the DOM methods.
```

Comment:
There's a misinterpretation here. The aim was to get the *code* not to use DOM methods. That way, document.getElementById doesn't need to be mocked.

Please also make your issue more descriptive and clear. Include more context and explanation so that it is self-explanatory, and point to the relevant code files which need to be fixed. Don't just file one-line issues. Your issue should enumerate all the tasks that need to be done in order to be properly complete. All the necessary context should be in the issue, don't force the user to click through to files in order to see what the issue is about.

Reply:
done